geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/fs/storage/parquet/ParquetFileSystemStorage.scala

```diff
@@ -93,7 +93,7 @@ object ParquetFileSystemStorage extends LazyLogging {
   }
 
   def validateParquetFile(file: Path): Unit = {
-    val reader = ParquetReader.builder(new GroupReadSupport(), file).build()
+    val reader = ParquetReader.builder(new SimpleFeatureReadSupport, file).build()
```

Comment:
i think we can stick with GroupReadSupport here - that way we will check that it's a valid parquet file, but we won't deserialize/manifest everything so it saves us some processing